man/corosync.conf.5

```diff
@@ -362,6 +362,11 @@ For real token timeout used by totem it's possible to read cmap value of
 .B runtime.config.totem.token
 key.
 
+The token_retransmit will be automatically calculated if token or token_retransmits_before_loss_const
```

Comment:
Only half true. `token_retransmit` will be always auto-calculated, no matter if token is set or not.